imperial_coldfront_plugin/templates/group_members.html

```diff
@@ -5,6 +5,10 @@ <h1>Group Members</h1>
     {% for member in group_members %}
       <li>
         {{ member.member.get_full_name }} - <a href="{% url 'imperial_coldfront_plugin:remove_group_member' member.pk %}">Remove access</a>
+        <a href="{% url 'imperial_coldfront_plugin:make_manager' member.pk %}">Make manager</a>
+        {% if member.is_manager %}
+          <a href="{% url 'imperial_coldfront_plugin:remove_manager' member.pk %}">Remove manager</a>
+        {% endif %}
```

Comment:
Given my below comment about preventing managers from adding/removing other managers, this only be visible to group owners.